tests/test_basicqc.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+import pyopenms as oms
+
+from qccalculator import basicqc, utils, idqc
+
+
+class MyTestCase(unittest.TestCase):
```

Comment:
can you make the test class name a bit more descriptive? Otherwise all good.

Reply:
Done